package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "eslint-plugin-instawork",
-  "version": "0.11.0",
+  "version": "0.12.0",
```

this should be done via `yarn release:minor` from master branch

```suggestion
  "version": "0.11.0",
```

---

Fixed it. Will run the release from master once this is merged.